src/common/legacy_config_opts.h

```diff
@@ -82,6 +82,7 @@ SAFE_OPTION(plugin_dir, OPT_STR)
 
 OPTION(compressor_zlib_isal, OPT_BOOL)
 OPTION(compressor_zlib_level, OPT_INT) //regular zlib compression level, not applicable to isa-l optimized version
+OPTION(compressor_zstd_level, OPT_INT) //regular Zstd compression level
```

Comment:
no need to define new settings in `src/common/legacy_config_opts.h`.

Reply:
I'll remove this then.

Reply:
I had to re-add this because otherwise the build fails with:
src/compressor/zstd/ZstdCompressor.h:32:45: error: 'class ConfigValues' has no member named 'compressor_zstd_level'; did you mean 'compressor_zlib_level'?